lxd/images.go

```diff
@@ -99,6 +99,13 @@ func unpack(file string, path string) error {
 	} else if strings.HasPrefix(extension, ".squashfs") {
 		command = "unsquashfs"
 		args = append(args, "-f", "-d", path, "-n")
+
+		mem, err := deviceTotalMemory()
+		mem = mem / 1024 / 1024 / 10
```

Comment:
I understand the intent and logic, but for someone coming along later I think the code will be quite confusing, especially since the commit message doesn't match the code.  2gb vs 2.5gb is nitpicking, but my initial reaction when looking at the code was "that's not 2gb".  I can see someone (like me) changing the code to actually compare against 2gb, thinking they're doing the right thing.